testcases/PowerNVDump.py

```diff
@@ -754,6 +766,31 @@ def runTest(self):
                 boot_type = self.kernel_crash(crash_type="hmc")
                 self.verify_dump_file(boot_type)
 
+class KernelCrash_KdumpSAN(PowerNVDump):
+
+    def runTest(self):
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.setup_test()
+        if self.distro == "rhel":
+            print("========== Deleting all partition(s) from %s ==========" % self.dev_path)
+            self.c.run_command("sfdisk --delete %s" % self.dev_path)
+            start, end = self.c.run_command("sfdisk -F %s |awk 'END{print $1;print $2}'" % self.dev_path)
+            self.c.run_command("echo \"%s1 : start=%s, size=%s, Id=83\" > /tmp/dd.layout" % (self.dev_path, start, end))
+            print("========== Create a single partition on %s ==========" % self.dev_path)
+            self.c.run_command("sfdisk --force %s < /tmp/dd.layout" % self.dev_path)
+            self.c.run_command("mkfs.xfs -f %s1" % self.dev_path)
+            self.c.run_command("sed -e '/^path/ i xfs %s1' -i /etc/kdump.conf;" % self.dev_path)
+            self.c.run_command("sed -i -e '$a%s1 /mnt xfs defaults 0 0' /etc/fstab" % self.dev_path)
+            self.c.run_command("mount /mnt && mkdir -p /mnt/var/crash")
+            self.c.run_command("systemctl restart kdump.service; sync", timeout=600)
+            boot_type = self.kernel_crash()
+            self.verify_dump_file(boot_type, dump_place="san")
+            self.c.run_command("umount /mnt")
+            self.c.run_command("sed -e '/^xfs/d' -i /etc/kdump.conf;")
+            self.c.run_command("sed '/\/mnt xfs/d' -i /etc/fstab")
+            self.c.run_command("systemctl restart kdump.service; sync", timeout=600)
+
+
```

Comment:
@kumudag if you have time can you make dump to happen on mulitple filesystems like say ext4, xfs, btrfs. If not please raise a issue as enhancements.

Reply:
This can be an enhancement once the basic functionality is working